modules/iam/src/main/scala/ch/epfl/bluebrain/nexus/commons/iam/IamClient.scala

```diff
@@ -25,8 +26,10 @@ trait IamClient[F[_]] {
     * Retrieve the ''caller'' form the optional [[OAuth2BearerToken]]
     *
     * @param optCredentials the optionally provided [[OAuth2BearerToken]]
+    * @param filterGroups   if true, will only return groups currently in use in IAM
+    *
     */
-  def getCaller(optCredentials: Option[OAuth2BearerToken]): F[Caller]
+  def getCaller(optCredentials: Option[OAuth2BearerToken], filterGroups: Boolean = false): F[Caller]
```

Comment:
I think in the trait it shouldn't have a default value. Maybe in the underlying implementation

Reply:
If you don't put the default in the trait, when you use `IamClient` somewhere, you won't be able to use the default value, as the trait  is the only thing that the compiler will know about.